_maps_2024/05-aki.md

```diff
@@ -0,0 +1,8 @@
+---
+day: 5
+creator: Akira Di Sandro
+title: My Winter Vacation '22-'23
+thumbnail: /assets/thumbnails/2024/05-Aki-journey.png
```

Comment:
@disandroa just FYI, I noticed that your image file has a capital J. I updated it in the `main` branch, but you might want to sync those changes back to your fork. See the note under step 1 [here](https://github.com/weitzman-musa/30DayMapChallenge?tab=readme-ov-file#participation-instructions).

On Windows, file capitalization doesn't matter, but it often does on Mac and Linux (and GH Pages runs Linux servers).